dali/python/nvidia/dali/backend.py

```diff
@@ -70,14 +70,26 @@ def check_cuda_runtime():
 
         if GetCufftVersion() == -1:
             deprecation_warning(
-                "Cannot access cuFFT library. Please check cuda installation and/or "
-                "if an appropriate wheel is installed.")
+                "nvidia-dali-cuda120 no longer ships with CUDA runtime. You need to install "
+                "it separately. Please check cuda installation and/or if an appropriate wheel "
+                "that provides cuFFT is installed. Please check "
+                "https://docs.nvidia.com/cuda/cuda-quick-start-guide/"
+                "index.html#pip-wheels-installation-linux "
+                "for the reference")
 
         if GetNppVersion() == -1:
-            deprecation_warning("Cannot access NPP library. Please check cuda installation and/or "
-                                "if an appropriate wheel is installed.")
-
+            deprecation_warning(
+                "nvidia-dali-cuda120 no longer ships with CUDA runtime. You need to install "
+                "it separately. Please check CUDA installation and/or if an appropriate wheel "
+                "that provides NPP is installed. Please check "
+                "https://docs.nvidia.com/cuda/cuda-quick-start-guide/"
```

Comment:
Similar suggestions as above (`"Please check"` phrase repeats at the beginning of sentences in this error message).

Reply:
Done